xrmocap/core/estimation/mview_sperson_smpl_estimator.py

```diff
@@ -366,26 +376,45 @@ def estimate_keypoints3d(self, cam_param: List[FisheyeCameraParameter],
                     keypoints3d, **optim_kwargs)
         return keypoints3d
 
-    def estimate_smpl(self, keypoints3d: Keypoints) -> SMPLData:
+    def estimate_smpl(self,
+                      keypoints3d: Keypoints,
+                      init_smpl_data: Union[None, SMPLData] = None,
+                      return_joints: bool = False,
+                      return_verts: bool = False) -> SMPLData:
         """Estimate smpl parameters according to keypoints3d.
 
         Args:
             keypoints3d (Keypoints):
                 A keypoints3d Keypoints instance, with only one person
                 inside. This method will take the person at
                 keypoints3d.get_keypoints()[:, 0, ...] to run smplify.
+            init_smpl_dict (dict, optional):
+                A dict of init parameters. init_dict.keys() is a
+                sub-set of self.__class__.OPTIM_PARAM.
+                Defaults to an empty dict.
+            return_joints (bool, optional):
+                Whether to return joints. Defaults to False.
+            return_verts (bool, optional):
+                Whether to return vertices. Defaults to False.
 
         Returns:
             SMPLData:
                 Smpl data of the person.
         """
         self.logger.info('Estimating SMPL.')
-        working_convention = 'smpl'
+        working_convention = self.smplify.body_model.keypoint_convention
         keypoints3d = convert_keypoints(
             keypoints=keypoints3d, dst=working_convention)
         keypoints3d = keypoints3d.to_tensor(device=self.smplify.device)
         kps3d_tensor = keypoints3d.get_keypoints()[:, 0, :, :3].float()
         kps3d_conf = keypoints3d.get_mask()[:, 0, ...]
+
+        # load init smpl data
+        if init_smpl_data is not None:
+            init_smpl_dict = init_smpl_data.to_tensor_dict()
```

Comment:
```suggestion
            init_smpl_dict = init_smpl_data.to_tensor_dict(device=self.smplify.device)
```

Reply:
Fixed in [b867ad1](https://github.com/openxrlab/xrmocap/pull/52/commits/b867ad1f23a1edab97d0111ff6943578a569535b).